docs/TranslationGuide.md

```diff
@@ -35,8 +35,8 @@ When adding new translatable strings to the GUI, they need to be added to each
 npm run i18n:extract
```

Comment:
Is it possible to limit this command to just update a single language file.

Reply:
I don't think so